### ISSUE TITLE
exception occured running transmission_main_example in anaconda/spyder

### ISSUE THREAD

**Issue by ggrammel** [Telecominfraproject/oopt-gnpy#99]
2018-08-01
Installing gnpy in Anaconda. Then navigating to examples directory and starting transmission_main_example.py from the toolbar creates the following output in IPython, throwing an exception:
```python
runfile('/projects/github/testing/gnpy/examples/transmission_main_example.py', wdir='/projects/github/testing/gnpy/examples')
    invalid souce node specified: '', replaced with Transceiver Site_A
    invalid destination node specified: '', replaced with Transceiver Site_B
    There are 4 network elements between Transceiver Site_A and Transceiver Site_B
    Transceiver Site_A
    Fiber Span1
      length (m): 80000.00
      loss (dB):  16.00
    Edfa Edfa1
      type_variety:      std_low_gain
      gain (dB):         16.00
      noise figure (dB): 7.12
      Power In (dBm):    3.83
      Power Out (dBm):   19.83
    Transceiver Site_B
      OSNR ASE (1nm):        34.83
      OSNR ASE (signal bw):  30.75
      SNR total (signal bw): 27.36
    An exception has occurred, use %tb to see the full traceback.

    SystemExit

    /anaconda3/lib/python3.6/site-packages/IPython/core/interactiveshell.py:2971: UserWarning: To exit: use 'exit', 'quit', or Ctrl-D.
      warn("To exit: use 'exit', 'quit', or Ctrl-D.", stacklevel=1)
```

looks like some explicit exit statement is required. Since the use of Anaconda is recommended, the issue should be fixed before updating master.

**Comment by dutc** [comment]
Can you provide more details about the exception that was encountered?

**Comment by ggrammel** [comment]
ran on a Mac from Anaconda interactive shell.

**Comment by dutc** [comment]
Can you give me the output of the command `env` in the environment where you're running this?

**Comment by dutc** [comment]
Fixed in version 1.0.
